<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alg_show</class>
 <widget class="QWidget" name="Alg_show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="1,4">
     <item row="0" column="1">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="Begin_Button">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>开始处理</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>实验信息</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>文件路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="FilePath_lineEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>算法选择</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,2">
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="5" column="1">
            <widget class="QComboBox" name="Reg_ComboBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <property name="text">
              <string>预处理</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="Classify_ComboBox"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_3">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <property name="text">
              <string>回归预测</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <property name="text">
              <string>降维算法</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>分类算法</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="Dimension_ComboBox"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="Pre_Button">
             <property name="text">
              <string>算法选择</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>数据选择</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="textBrowser"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
